crystal-mode.el

```diff
@@ -2304,144 +2304,149 @@ See `font-lock-syntax-table'.")
     (,(concat
        crystal-font-lock-keyword-beg-re
        (regexp-opt
-        '("alias"
+        '("abstract"
+          "alias"
+          "annotation"
+          "as"
+          "as?"
+          "asm"
           "begin"
           "break"
           "case"
           "class"
-          "context"
           "def"
-          "describe"
-          "defined?"
           "do"
-          "elsif"
           "else"
-          "fail"
+          "elsif"
+          "end"
           "ensure"
+          "enum"
+          "extend"
+          "false"
           "for"
-          "end"
+          "fun"
           "if"
           "in"
-          "it"
+          "include"
+          "instance_sizeof"
+          "is_a?"
```

Comment:
I'm not 100% sure we should include methods as keywords when they aren't toplevel.  `is_a?` is a method on `Object`.

Reply:
Check my another reply for answer.

Maybe we should add all methods defined on Object into non-keyword group, to make it highlight for user do not occasionally define methods with the same name.